timeseries/tests/smoketests/test_all_models.py

```diff
@@ -55,20 +54,24 @@ def generate_train_and_test_data(
     return train_data, test_data
 
 
-DUMMY_MODEL_HPARAMS = {"max_epochs": 1, "num_batches_per_epoch": 1, "use_fallback_model": False}
+DUMMY_MODEL_HPARAMS = {
+    "max_epochs": 1,
+    "num_batches_per_epoch": 1,
+    "use_fallback_model": False,
+}
 
 ALL_MODELS = {
     "ADIDA": DUMMY_MODEL_HPARAMS,
     "Average": DUMMY_MODEL_HPARAMS,
-    "AutoCES": DUMMY_MODEL_HPARAMS,
+    # "AutoCES": {**DUMMY_MODEL_HPARAMS, "model": "S"},
```

Comment:
Should we remove this line completely instead of commenting out?

Reply:
If we are going for the approach "some models are basically exchangeable when it comes to how they handle covariates", we can only keep one representative from each group in these tests. For example, ADIDA, IMAPA, Croston -> only keep Croston.

Reply:
ADIDA, IMAPA, etc. are very fast to run, so will leave them in for now.